extension/src/plots/paths/tree.ts

```diff
@@ -32,7 +34,19 @@ export class PlotsPathsTree extends BasePathSelectionTree<WorkspacePlots> {
     )
   }
 
-  public getRepositoryChildren(dvcRoot: string, path: string | undefined) {
+  protected getBaseTreeItem({
+    dvcRoot,
+    path,
+    collapsibleState
+  }: PathSelectionItem) {
+    const resourceUri = getDecoratableUri(
+      join(dvcRoot, path),
+      DecoratableTreeItemScheme.PLOTS
+    )
+    return new TreeItem(resourceUri, collapsibleState)
```

Comment:
[F] By providing a "decoratable" Uri here with the `dvc.plots` scheme we won't run into the SCM decorations and we'll be able to provide a color when there is an error for that plots. See `extension/src/experiments/model/decorationProvider.ts` for a working example. 

The same approach does not work for the experiments column tree as the path can be `:` separated. Hence they have been left separate.